lib/buildpacks-ci-pipeline-updater.rb

```diff
@@ -100,6 +99,10 @@ def parse_args(args)
       opts.on("--template=TEMPLATE", "-tTEMPLATE", "Only update pipelines from the specified template") do |template_string|
         specified_options[:template] = template_string
       end
+
+      opts.on("--stack=STACK", "-sSTACK", "Use the specified stack config") do |stack_string|
+        specified_options[:stack] = stack_string
+      end
     end
     opt_parser.parse!(args)
```

Comment:
It looks like `--stack` is a mandatory "option", so maybe check that it has been specified:

```
if specified_options[:stack].nil?
   raise "Missing --stack option"
end
```

Reply:
Good idea! Fixed in 8ed5cd9f

Reply:
Actually, this needs more thought as it only applies to the `binary-builder` pipeline, not the others. I'll move it to somewhere I can check for `binary-builder`

Reply:
Added a `binary-builder` specific check in aa5329d947c4e1ee3b2fb661eddaf4bd59c86a27